tests/conftest.py

```diff
@@ -174,16 +182,160 @@ def model_with_submodel(model_class_with_submodel, data_class):
 
     contract = dict(inputs=dict(B='test'))
     sub_contract = dict(
-        kwargs=dict(a='d'),
-        nets=dict(net='net2'),
-        inputs=dict(A='test')
-    )
+        kwargs=dict(a='d'), nets=dict(net='net2'), inputs=dict(A='test'))
 
-    model = model_class_with_submodel(sub_contract=sub_contract,
-                                      contract=contract)
+    model = model_class_with_submodel(
+        sub_contract=sub_contract, contract=contract)
     model._data = data
     model.submodel._data = data
 
     model.kwargs.update(**kwargs)
 
     return model
+
+
+@pytest.fixture
+def args():
+    from argparse import Namespace
+    import torch.nn as nn
+    return Namespace(
+        classifier_args={'dropout': 0.2},
+        classifier_type='convnet',
+        clean=False,
+        command=None,
+        config_file=None,
+        criterion=nn.CrossEntropyLoss(),
+        device=0,
```

Comment:
Aren't these from args_mock? Is this file not being used?

This part needs to be changed to 'cpu':
https://github.com/rdevon/cortex/blob/master/tests/args_mock.py#L11

Otherwise it will fail on machines with GPU (in my tests.)